packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -25,8 +25,9 @@ const FilesManager: FunctionComponent<Props> = ({
     full: 0,
   },
 }) => {
-  const { full, free } = memorySpace
-  const systemMemory = full - free
+  const totalMemorySpace = 16000000000
```

Comment:
🟡
The value should be fetched from OS, ultimately please move this mock value to packages/app/src/backend/adapters/pure-phone-storage/pure-phone-storage.adapter.ts

Reply:
The value moved in [15bba30](https://github.com/mudita/mudita-center/pull/598/commits/15bba3011d4d8c07a975fa30c1f98de92bc025db)